ppr-ui/src/composables/mhrInformation/useMhrInformation.ts

```diff
@@ -85,22 +91,23 @@ export const useMhrInformation = () => {
       transferDate: getMhrTransferDate.value,
       ownLand: getMhrTransferOwnLand.value,
       attentionReference: getMhrTransferAttentionReference.value,
-      documentDescription: 'SALE OR GIFT',
+      documentDescription: UIRegistrationTypes.TRANSFER_OF_SALE,
       submittingParty: {
+        // Need to confirm if this needs to be a Business Name instead
+        // because in UI, Submitting Party is a business not a person
+        // but the API spec requires firstName, lastName
         personName: {
           first: getCurrentUser.value.firstname,
           last: getCurrentUser.value.lastname
         },
-        address: {
-          street: '222 SUMMER STREET',
-          city: 'VICTORIA',
-          region: 'BC',
-          country: 'CA',
-          postalCode: 'V8W 2V8'
-        },
-        emailAddress: getCurrentUser.value.contacts[0].email,
-        phoneNumber: getCurrentUser.value.contacts[0].phone.replace(/[^0-9.]+/g, '') // Remove special chars
-      },
+        address: getMhrTransferSubmittingParty.value.address,
+        emailAddress: getMhrTransferSubmittingParty.value.emailAddress,
+        phoneNumber: getMhrTransferSubmittingParty.value.phoneNumber,
+        phoneExtension: getMhrTransferSubmittingParty.value.phoneExtension
+        // Cleanup below after confirming that Submitting Party info is Account's Admin info
+        // emailAddress: getCurrentUser.value.contacts[0].email,
+        // phoneNumber: getCurrentUser.value.contacts[0].phone.replace(/[^0-9.]+/g, '') // Remove special chars
```

Comment:
I think you can confirm with Tracey and remove these comments 👍 
That being said, there is no submitting party information completed in a Transfer filing so how are the root properties of `getMhrTransferSubmittingParty` populated? 
Is it current account information?

Reply:
Ah i looked below and see. 
So isn't saving the results of `getAccountInfoFromAuth` ie currentAccount to both current account info and submitting party info a bit of a decoy?

Reply:
Or is that different from current account?

Reply:
Current account is the logged-in user (e.g BCREG2 Della FORTYTHREE). The Account Info contains the account name (e.g. PPR 2). So they are not the same. We are making a separate call to `/orgs` to get name and address info. We are doing some parsing to align the prop names of the interfaces. We could probably optimize it still.

Reply:
For sure, the `/orgs` request is part of `getAccountInfoFromAuth` right?
That is called in `MhrInformation` via `getSubmittingPartyInformation`, saved to localState and subsequently submitting party info. So what do we designate that? 
It's misleading as the CONST used in that request is `CurrentAccount`

Reply:
So to be plain, Submitting Party is or is not the Current Account?

Reply:
Just to reply to your comment. Based on **currentAccount** id we make a calls to `/orgs` to get Account Info submitting party.